moped-editor/src/views/projects/newProjectView/NewProjectMap.js

```diff
@@ -52,14 +78,36 @@ export const useStyles = makeStyles({
     height: 50,
     position: "absolute",
     alignItems: "center",
-    right: 32,
+    width: "21rem",
+    left: "1rem",
+    top: ".5rem",
+    zIndex: 2,
     // Keep geocoder input in set position when mapbox-gl-geocoder.css media queries kick in
     "@media (max-width:640px)": {
       top: 32,
     },
   },
+  exampleWrapper: {
+    position: "relative",
+    marginTop: theme.spacing(3),
+    height: 380,
+  },
+  speedDial: {
+    right: "3.5rem !important",
+    bottom: "4.5rem !important",
+    position: "absolute",
+    zIndex: 1,
+    "&.MuiSpeedDial-directionUp, &.MuiSpeedDial-directionLeft": {
+      bottom: theme.spacing(2),
+      right: theme.spacing(2),
+    },
+    "&.MuiSpeedDial-directionDown, &.MuiSpeedDial-directionRight": {
+      top: theme.spacing(2),
+      left: theme.spacing(2),
+    },
+  },
   ...layerSelectStyles,
-});
+}));
```

Comment:
Extensive styling given the level of detail and overrides to make the map stretch, the search bar work, etc.